.ci/lib.sh

```diff
@@ -48,15 +48,45 @@ function clone_build_and_install() {
 	popd
 }
 
+function install_yq() {
+	GOPATH=${GOPATH:-${HOME}/go}
+	mkdir -p "${GOPATH}/bin"
```

Comment:
niggle: maybe do the arch check before the mkdir?

Reply:
to avoid create the dir if the arch is not supported?

Reply:
yeah, that's all - if the arch is not supported, no point making the dir in the first place :-)

Reply:
fair enough, moved.